zerver/lib/events.py

```diff
@@ -204,6 +205,13 @@ def fetch_initial_state_data(
         # remove this parameter from the API.
         state["max_message_id"] = max_message_id_for_user(user_profile)
 
+    if want("saved_replies"):
+        if user_profile is None:
+            state["saved_replies"] = []
+        else:
+            saved_replies = SavedReply.objects.filter(user_profile=user_profile)
+            state["saved_replies"] = [saved_reply.to_dict() for saved_reply in saved_replies]
```

Comment:
We might need to specify an explicit `to_api_dict` function after my `models.py` comments are done to only include the fields that we want in the API.

Reply:
Also this block should call `do_get_saved_replies` rather than duplicating it; that approach sets us up for success in future refactoring. The above comment still applies to the `do_get_saved_replies` implementation.

Reply:
Except the above comment about `to_api_dict` is nonsense because `to_dict` is a function defined only for this purpose :). `to_api_dict` might be a nicer name though.